Makefile

```diff
@@ -56,6 +56,7 @@ LOGCHECK := $(TOOLS_GOBIN_DIR)/$(LOGCHECK_BIN)-$(LOGCHECK_VER)
 export LOGCHECK # so hack scripts can use it
 
 ARCH := $(subst 64,,$(shell uname -p | sed s/x86_/amd/))64
+OS := linux
```

Comment:
won't this break Mac ?

Reply:
it breaks make on my Mac, doesn't it?

Reply:
I have updated OS to have empty value by default. It can be overwritten with the required value in test workflow. Now it should not break on Mac.